src/presentation/http/router/note.ts

```diff
@@ -54,10 +56,7 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
     Params: {
       notePublicId: NotePublicId;
     },
-    Reply: {
-      note : Note
-      accessRights: { canEdit: boolean },
-    } | ErrorResponse,
+    Reply: NoteResponse| ErrorResponse,
```

Comment:
You've changed a generic that is used for TypeScript only. It was ok. I meant adding a Fastify-schema for serialisation that will be used in runtime. See https://fastify.dev/docs/latest/Guides/Getting-Started/#serialize-your-data

Check the `/myself` route handler for example